modules/populateFunction.js

```diff
@@ -0,0 +1,19 @@
+import tasks from './objects.js';
+
+const holder = document.getElementById('list-holder');
+
+const populate = () => {
+  tasks.forEach((element) => {
+    const add = document.createElement('div');
+    add.dataset.elementid = element.id;
+    add.innerHTML = `
+  <div class="container">
+  <span><input type='checkbox'>${element.string}</span>
+  <div><i class="fa-solid fa-ellipsis-vertical"></i></div>
+  </div><hr>
```

Comment:
- [ ] I would suggest you please make sure to populate the HTML list item element for each task. You have populated the string but make sure to populate the `completed [bool]`. For reference:
``` <p>${completed ? 'completed' : 'not completed'}</p>`; ```